lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -1017,6 +1017,10 @@ void ResourceMap::loadDefaultConfiguration()
   addAsUnsignedInteger("SobolIndicesAlgorithm-DefaultBlockSize", 1);
   addAsUnsignedInteger("SobolIndicesAlgorithm-DefaultBootstrapSize", 100);
 
+  // RatioOfUniforms //
+  addAsString("RatioOfUniforms-OptimizationAlgorithm", "Cobyla");
+  addAsUnsignedInteger("RatioOfUniforms-CandidateNumber", 10000);
```

Comment:
Given that this is used as the sample size of a Sobol' sequence, it might be interesting to select a base-2 integer, e.g. $2^{13} =8192$ or $2^{14}=16384$.

Reply:
True! It has been changed to 8192.